src/importexport/musicxml/internal/musicxmlconfiguration.cpp

```diff
@@ -45,6 +46,9 @@ void MusicXmlConfiguration::init()
     settings()->setDefaultValue(MUSICXML_EXPORT_LAYOUT_KEY, Val(true));
     settings()->setDefaultValue(MUSICXML_EXPORT_BREAKS_TYPE_KEY, Val(MusicxmlExportBreaksType::All));
     settings()->setDefaultValue(MUSICXML_EXPORT_INVISIBLE_ELEMENTS_KEY, Val(false));
+    settings()->setDescription(MUSICXML_EXPORT_INVISIBLE_ELEMENTS_KEY, muse::qtrc("project/export",
```

Comment:
It should be "iex_musicxml", not "project/export". You should also use muse::trc here

Reply:
done

Reply:
doesn't work. See also a few lines up, using the same qtrc and the same context

Reply:
hold on, need to remove the `toStdString()` for the trc to work. 
Will change the other one too, as well the the context for both.